app/assets/stylesheets/custom.css.scss

```diff
@@ -302,3 +300,68 @@ table td {
     margin-right: 1em;
   }
 }
+
+.index-container img {
+  text-align: center;
+  padding-top: 4px;
+}
+
+.index-container h4 {
+  min-height: 50px;
+}
+
+.index-container h2 {
+  text-align: center;
+}
+
+.description {
+  text-align: center;
+  font-size: 20px;
+}
+
+.thumbnail {
+  width: 300px;
+  margin-left: 33%;
+  opacity: 0.7;
+  background-color: cornsilk;
+  text-align: center;
+}
+
+.checkouts-page  {
+  margin-left: 25%;
+}
+
+.checkouts-page h2 {
+  text-align: center;
+  margin-left: -375px;
+}
+
+.confirmation  {
+  margin-left: 25%;
+}
+
+.confirmation h2 {
+  text-align: center;
+  margin-left: -375px;
+}
+
+.flashes {
+  margin-left: 35%;
+}
+
+.user-show {
+  width: 300px;
```

Comment:
Properties should be ordered background-color, margin-left, opacity, text-align, width